src/core/utils/Renderer.ts

```diff
@@ -212,6 +212,18 @@ export class Renderer {
             // Increment the offset to be positioned after the reference node.
             location[1] += 1;
         }
+        // When targetting the first of two <br> in an element that has only
+        // that, we need to correct the offset because `VDocumentMap` has no way
+        // to know, given that VDocument has one line break to represent these
```

Comment:
That would require quite a change... `VDocumentMap.toDom` would need to be able to return `Node[]` instead of `[Node, number]` (which would be good in any case if we plan on having more cases where one `VNode` represents several Nodes). That in turn implies changing `VDocumentMap.toDomLocation` and `VDocumentMap.set`. We can "let it know, let it know" :notes: in the `Parser` but we need to do it again in the `Renderer`. Naively checking if the pair already exists like for `fromDom` causes issues in this case because we risk comparing different versions of the same DOM Node. Then use of `toDomLocation` in the `Renderer` has to be adapted, as well as `_locate` in `EventManager`. Is this all worth it, just to avoid my 8-line fix?